tests/unit/unit1459.c

```diff
@@ -0,0 +1,42 @@
+/***************************************************************************
+ *                                  _   _ ____  _
+ *  Project                     ___| | | |  _ \| |
+ *                             / __| | | | |_) | |
+ *                            | (__| |_| |  _ <| |___
+ *                             \___|\___/|_| \_\_____|
+ *
+ * Copyright (C) 1998 - 2019, Daniel Stenberg, <daniel@haxx.se>, et al.
+ *
+ * This software is licensed as described in the file COPYING, which
+ * you should have received as part of this distribution. The terms
+ * are also available at https://curl.haxx.se/docs/copyright.html.
+ *
+ * You may opt to use, copy, modify, merge, publish, distribute and/or sell
+ * copies of the Software, and permit persons to whom the Software is
+ * furnished to do so, under the terms of the COPYING file.
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTY OF ANY
+ * KIND, either express or implied.
+ *
+ ***************************************************************************/
+#include "curlcheck.h"
+
+#include <curl/curl.h>
+
+UNITTEST_START
+
+int rc;
+
+rc = _curl_is_slist_option(CURLOPT_HTTPHEADER);
+fail_unless(rc != 0, "CURLOPT_HTTPHEADER is an slist option");
+
+rc = _curl_is_string_option(CURLOPT_COOKIE);
+fail_unless(rc != 0, "CURLOPT_COOKIE is a string option");
+
+rc = _curl_is_long_option(CURLOPT_TIMEOUT);
+fail_unless(rc != 0, "CURLOPT_TIMEOUT is a long option");
+
+rc = _curl_is_off_t_option(CURLOPT_MAX_SEND_SPEED_LARGE);
+fail_unless(rc != 0, "CURLOPT_MAX_SEND_SPEED_LARGE is an off_t option");
+
+UNITTEST_STOP
```

Comment:
You need to add the unit test build to `Makefile.inc` in tests/unit to make it build. Something like:
~~~diff
diff --git a/tests/unit/Makefile.inc b/tests/unit/Makefile.inc
index f3cba1c2a..28e6d5fe4 100644
--- a/tests/unit/Makefile.inc
+++ b/tests/unit/Makefile.inc
@@ -7,10 +7,11 @@ UNITFILES = curlcheck.h \
 # These are all unit test programs
 UNITPROGS = unit1300 unit1301 unit1302 unit1303 unit1304 unit1305 unit1307 \
  unit1308 unit1309 unit1323 \
  unit1330 unit1394 unit1395 unit1396 unit1397 unit1398 \
  unit1399 \
+ unit1459 \
  unit1600 unit1601 unit1602 unit1603 unit1604 unit1605 unit1606 unit1607 \
  unit1608 unit1609 unit1620 unit1621 \
  unit1650 unit1651 unit1652 unit1653 unit1654
 
 unit1300_SOURCES = unit1300.c $(UNITFILES)
@@ -65,10 +66,13 @@ unit1398_SOURCES = unit1398.c $(UNITFILES)
 unit1398_CPPFLAGS = $(AM_CPPFLAGS)
 
 unit1399_SOURCES = unit1399.c $(UNITFILES)
 unit1399_CPPFLAGS = $(AM_CPPFLAGS)
 
+unit1549_SOURCES = unit1549.c $(UNITFILES)
+unit1549_CPPFLAGS = $(AM_CPPFLAGS)
+
 unit1600_SOURCES = unit1600.c $(UNITFILES)
 unit1600_CPPFLAGS = $(AM_CPPFLAGS)
 
 unit1601_SOURCES = unit1601.c $(UNITFILES)
 unit1601_CPPFLAGS = $(AM_CPPFLAGS)